kart/json_diff_writers.py

```diff
@@ -98,6 +107,11 @@ def default(self, obj):
                 result["feature"] = self.filtered_ds_feature_deltas_as_json(
                     ds_path, ds_diff
                 )
+            if "tiles" in ds_diff:
```

Comment:
we have "meta" (ambiguous) and "feature" (singular), should we have "tile" singular too for consistency

Reply:
I'll do this in a separate commit.